api/lib/gateways/Academy-Benefits/FetchRecord.js

```diff
@@ -5,6 +5,7 @@ const { fetchCustomerBenefitsSQL, fetchCustomerHouseholdSQL } = loadSQL(
 );
 const { IncomeFrequency } = require('../../Constants');
 const isEqual = require('lodash/isEqual');
```

Comment:
Looks like we might not need to include `isEqual` anymore (unless I'm missing a reference to it somewhere below). 🙂

Reply:
That has already been removed